app/assets/javascripts/activeadmin_addons/select2.js

```diff
@@ -110,9 +115,13 @@ $(function() {
             query.q[parent + '_eq'] = parentId;
           }
 
+          if (per_page) {
+            query.per_page = per_page;
+          }
+
           return query;
         },
-        results: function(data, page) {
+        processResults: function(data, params) {
```

Comment:
Expected method shorthand                                        object-shorthand<br>Missing space before function parentheses                        space-before-function-paren<br>'params' is defined but never used                               no-unused-vars